internal/dag/builder_test.go

```diff
@@ -3685,10 +3662,7 @@ func TestDAGIngressRouteUniqueFQDNs(t *testing.T) {
 
 	for name, tc := range tests {
 		t.Run(name, func(t *testing.T) {
-			b := Builder{
-				ExternalInsecurePort: 10080,
-				ExternalSecurePort:   10443,
-			}
+			b := Builder{}
```

Comment:
nit, you can write 
```go
var b Builder
```
This is perhaps a matter of taste but I use the `var` form to indicate to the reader that we're declaring a `Builder` without explicitly initalising it. This leaves the `Builder{ ... }` form for when explicit initialisation of `Builder`'s fields is needed.

Reply:
I read this in one of your blogs, ( 😄 ) will do.